source/switching_to_live/index.html.md.erb

```diff
@@ -41,12 +41,16 @@ You can still use your test account.
 
 ## 2. Connect your live account to your PSP
 
-Once you have a GOV.UK Pay live account, you can connect it to:
+Once you have a GOV.UK Pay live account, you can connect that account to your PSP. You can use either of the following PSPs with GOV.UK Pay:
 
-- [Stripe](/switching_to_live/set_up_a_live_stripe_account/#connect-your-live-account-to-stripe)
-- [ePDQ](/switching_to_live/set_up_a_live_epdq_account/#connect-your-live-account-to-epdq)
-- [SmartPay](/switching_to_live/set_up_a_live_smartpay_account/#connect-your-live-account-to-smartpay)
-- [Worldpay](/switching_to_live/set_up_a_live_worldpay_account/#connect-your-live-account-to-worldpay)
+- GOV.UK Pay's PSP, [Stripe](/switching_to_live/set_up_a_live_stripe_account/#connect-your-live-account-to-stripe)
+- if you are a central government or health sector organisation, Government Banking's PSP, [Worldpay](/switching_to_live/set_up_a_live_worldpay_account/#connect-your-live-account-to-worldpay)
+
+If you use WorldPay and Government Banking decides to change PSP in future, we will work with you at that time to move you on to the new PSP.
+
+Effective from DATE, you cannot use GOV.UK Pay with any other PSPs.
+
+If you agreed with GOV.UK Pay before this date to use either [ePDQ](/switching_to_live/set_up_a_live_epdq_account/#connect-your-live-account-to-epdq) or [SmartPay](/switching_to_live/set_up_a_live_smartpay_account/#connect-your-live-account-to-smartpay), you can connect your live account to that PSP.
```

Comment:
Do we need to add anything like this at the top of this page: https://docs.payments.service.gov.uk/switching_to_live/set_up_a_live_epdq_account/#connect-your-live-account-to-epdq and this page: https://docs.payments.service.gov.uk/switching_to_live/set_up_a_live_smartpay_account/#connect-your-live-account-to-smartpay (sorry it might be in this list and i've missed it)

Reply:
good point, will add